ts/components/staking/header/sub_menu.tsx

```diff
@@ -167,21 +167,12 @@ export const SubMenu = (props: ISubMenuProps) => {
                     setActiveAccount(null);
                 }
             });
-        } else if (walletConnector && walletConnector.name === 'walletlink') {
-            setIconName(constants.PROVIDER_TYPE_TO_ICON.COINBASE_WALLET);
-        } else if (walletConnector && walletConnector.name === 'walletconnect') {
-            setIconName(constants.PROVIDER_TYPE_TO_ICON.WALLET_CONNECT);
         }
     }, []);
 
     const logout = async () => {
         if (typeof window !== undefined) {
-            window.localStorage.removeItem('__WalletLink__:https://www.walletlink.org:Addresses');
-            window.localStorage.removeItem('__WalletLink__:https://www.walletlink.org:SessionId');
-            window.localStorage.removeItem('-walletlink:https://www.walletlink.org:session:id');
-            window.localStorage.removeItem('-walletlink:https://www.walletlink.org:session:secret');
-            window.localStorage.removeItem('-walletlink:https://www.walletlink.org:session:linked');
-            window.localStorage.removeItem('WALLET_CONNECTOR');
+            window.localStorage.clear();
```

Comment:
I don't think we can just clear the entire localStorage, that is fairly destructive and there might be other important information we want to keep there even if the user logs out of the staking porta.